pglookout/pglookout.py

```diff
@@ -813,10 +814,22 @@ def delete_alert_file(self, filename):
     def within_dbpoll_time(self, time1, time2):
         return abs((time1 - time2).total_seconds()) < self.config.get("db_poll_interval", 5.0)
 
+    def _check_cluster_monitor_thread_health(self, now: float) -> None:
+        health_timeout_seconds = self.config.get("cluster_monitor_health_timeout_seconds")
```

Comment:
A default value of twice the value of `replication_state_check_interval` would enable measurement at a reasonable interval by default without extra config effort for users, I think it would be better than being disabled by default.